docs/pages/api-docs/x-grid.md

```diff
@@ -84,7 +84,8 @@ Api of the `components` props of type `GridSlotsComponent`
 | <span class="prop-name">ErrorOverlay</span> | <span class="prop-type">React.ElementType&lt;BaseComponentProps & ErrorOverlayProps></span> | <span class="prop-type">ErrorOverlay</span> | Error overlay component rendered above the grid when an error is caught.|
 | <span class="prop-name">FilterPanel</span> | <span class="prop-type">React.ElementType&lt;BaseComponentProps></span> | <span class="prop-type">FilterPanel</span> | Filter panel component rendered when clicking the filter button.|
 | <span class="prop-name">Footer</span> | <span class="prop-type">React.ElementType&lt;BaseComponentProps></span> | <span class="prop-type">GridFooter</span> | Footer component rendered at the bottom of the grid viewport.|
-| <span class="prop-name">Header</span> | <span class="prop-type">React.ElementType&lt;BaseComponentProps></span> | <span class="prop-type">GridHeader</span> | Header component rendered above the grid column header bar.|
+| <span class="prop-name">Toolbar</span> | <span class="prop-type">React.ElementType&lt;BaseComponentProps></span> | <span class="prop-type">GridToolbar</span> | Toolbar component rendered above the grid column header bar.|
+| <span class="prop-name">PreferencesPanel</span> | <span class="prop-type">React.ElementType&lt;BaseComponentProps></span> | <span class="prop-type">PreferencesPanel</span> | PreferencesPanel component rendered above the grid toolbar component slot.|
```

Comment:
`PreferencesPanel component rendered above the grid toolbar component slot.`
It's not rendered above. 
Component that render the ColumnSelector or FilterPanel within a Panel component.

Reply:
Done